docs/getting-started/contributors.md

```diff
@@ -132,5 +132,10 @@ copy everything from the default config to the local config.
 `app-config.local.yaml` will be merged with `app-config.yaml` and overwrite the
 default app configs.
 
+> NOTE: If you want to add your own configuration values to access in the
```

Comment:
Don't we have support for some kind of admonition / alert kinda thing for this type of block? Would be very nice to have note and warning at least, nicely styled

Reply:
Docusaurus has something like that, but I wonder if that is already available <2.0.0 https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions

Reply:
![image](https://user-images.githubusercontent.com/648527/109271622-15791a00-7810-11eb-9eba-769a0d77eb1e.png)

I would interpret that as: no 😆

Reply:
Aight, but at least the `>` (as his code already uses, and as seen in your screenshot) stands out as well